### ISSUE TITLE
[BUG] - Strange positioning of the font styling list popup when clicking on the dropdown

### ISSUE THREAD

**Issue by vbaxan** [tneotia/html-editor-enhanced#160]
This is how it looks like when clicking on the dropdown with "Normal" and other styling options:
<img width="334" alt="Screenshot 2021-11-05 at 09 45 38" src="https://user-images.githubusercontent.com/64584645/140476220-b253594e-6692-43c1-b176-c136f80a00a4.png">

Is there something that has to be set maybe, in order for the popup to get displayed properly?

**Comment by tneotia** [comment]
Are you using the custom toolbar position here? If so that might be causing it but I may be able to fix it.

**Comment by vbaxan** [comment]
Yes, but I think I've seen this using `ToolbarPosition.belowEditor` as well.

**Comment by istidanaharjanti** [comment]
hi! how to fix this issue?

**Comment by tneotia** [comment]
@istidanaharjanti and @vbaxan can you both add `dropdownMenuDirection: DropdownMenuDirection.up,` to the `HtmlToolbarOptions` in your implementation? Make sure to add inside both `ToolbarWidget` and `Html` incase you are using custom toolbar position.

**Comment by vbaxan** [comment]
@tneotia no, does not work for me - I still have this effect as in the image.

**Comment by tneotia** [comment]
Could you please send the page widget code so I can try and reproduce it?

**Comment by 1136346879** [comment]
> dropdownMenuDirection: DropdownMenuDirection.up

hello，Are you done with that?  I do the same thing now, one moment it looks normal, the next moment it looks like the image above ，If yes, how did you solve it？
